File: Backend/DTOs/Draw/DrawDrawingDTO.cs
namespace Learnz.DTOs;
public class DrawDrawingDTO
{
    public List<DrawPageGetDTO> Pages { get; set; }
    public string Name { get; set; }
    public bool Editable { get; set; }
    public string? NewUserMakingChangesName { get; set; }
}

File: Frontend/src/app/DTOs/Draw/DrawDrawingDTO.ts
import { DrawCanvasStorageDTO } from "./DrawCanvasStorageDTO";
import { DrawPageGetDTO } from "./DrawPageGetDTO";

export interface DrawDrawingDTO {
    pages: DrawPageGetDTO[];
    drawSegmensts: DrawCanvasStorageDTO[] | null;
    name: string;
    editable: boolean;
    newUserMakingChangesName: string | null;
}